cmd/controller-bootstrap/command/root.go

```diff
@@ -62,36 +62,37 @@ func init() {
 	}
 	defaultTemplatesDir = filepath.Join(cd, "templates")
 
-	rootCmd.PersistentFlags().StringVar(
-		&optServiceAlias, "aws-service-alias", "", "AWS service alias",
-	)
-	rootCmd.PersistentFlags().StringVar(
+	templateCmd.PersistentFlags().StringVar(
 		&optRuntimeVersion, "ack-runtime-version", "", "Version of aws-controllers-k8s/runtime",
 	)
```

Comment:
A more idiomatic way of constructing command-line interfaces is to make required things as `arguments` and non-required things as `options` (or `flags`).

You might want to consider making parameters that are required be arguments to the `generate` command instead of flags.

Read more here: https://github.com/spf13/cobra/blob/master/user_guide.md#positional-and-custom-arguments

You can do this in a future PR, however. Not required to change anything here.

Reply:
Thank you for pointing this out! I will address in a future PR to make parameters that are required to be arguments to the `generate` command.